examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -260,7 +264,8 @@ class ClusterCommand : public InteractionModelCommands, public ModelCommand, pub
 private:
     chip::ClusterId mClusterId;
     chip::CommandId mCommandId;
-    chip::ScopedNodeId mScopedNodeId;
+    chip::ScopedNodeId mPeerNodeId;
+    chip::ScopedNodeId mCheckInNodeId;
```

Comment:
These members need to be documented in terms of how they are expected to be used.  Right now we are leaking them across command invocations, which is very much not good.

Reply:
updated documentation, thanks